src/components/SignUp/CareerInfo.js

```diff
@@ -165,11 +169,31 @@ class CareerInfo extends Component {
       if (this.props.fromEdit) {
         Actions.pop();
       } else {
-        Actions.expertInfo({me: this.props.me});
+        Actions.expertInfo({ me: this.props.me });
       }
     }
   }
 
+  onGetCareerCallback(result, error) {
+    if (error) {
+      alert(JSON.stringify(error));
+    } else if (result.length !== 0) {
+      this.state.option[1] = CareerData.role[CareerData.area.indexOf(result[0].area)].list;
+
+      this.state.selected[0] = result[0].area;
+      this.state.selected[1] = result[0].role;
+      this.state.selected[2] = result[0].years;
+      this.state.selected[3] = result[0].education_background;
+
+      this.state.checked[0] = true;
+      this.state.checked[1] = true;
+      this.state.checked[2] = true;
+      this.state.checked[3] = true;
+
+      this.forceUpdate();
+    }
+  }
```

Comment:
You can use for statement

for (let i = 0; i < 4; i += 1)
  this.state.checked[i] = true;

Also, I think you can probably find more efficient way.